src/util/extract-custom-separator.js

```diff
@@ -0,0 +1,13 @@
+import { SEPARATOR } from '../constant';
+
+function extractCustomSeparator(input) {
+  const customMatch = input.match(SEPARATOR.CUSTOM);
+  if (customMatch) {
+    const separator = customMatch[1];
+    const inputWithoutSeparator = input.replace(customMatch[0], '');
```

Comment:
Airbnb 코드 컨벤션에 나와 있는 것처럼, match의 반환값을 구조 분해 할당으로 받아오는 건 어떨까요? 
[destructuring--array](https://github.com/airbnb/javascript?tab=readme-ov-file#destructuring--array)
```suggestion
const [matchedString, separator] = customMatch;
```

Reply:
좋은 피드백 감사합니다!